plasmapy/atomic/atomic.py

```diff
@@ -3,16 +3,24 @@
 import numpy as np
 import re
 from astropy import units as u, constants as const
-from warnings import warn
+import warnings
 from .elements import atomic_symbols, atomic_symbols_dict, Elements
 from .isotopes import Isotopes
 
-
+"""
+__all__ = ['atomic_symbol', 'isotope_symbol', 'atomic_number',
+           'is_isotope_stable', 'half_life', 'mass_number',
+           'element_name', 'standard_atomic_weight', 'isotope_mass',
+           'ion_mass', 'known_isotopes', 'common_isotopes',
+           'stable_isotopes', 'isotopic_abundance', 'charge_state',
+           'electric_charge']
+"""
```

Comment:
I suppose this is here simply for debugging purposes? Am I wrong? Is this somehow necessary for automodapi?

Reply:
Woops, this shouldn't be here!